frontend/packages/core/src/AppLayout/header.tsx

```diff
@@ -31,15 +22,11 @@ const MenuButton = styled(IconButton)`
 `;
 
 const Title = styled(Typography)`
-  margin-right: 25px;
-  font-weight: bolder;
-`;
-
-const Divider = styled(MuiDivider)`
-  ${({ theme }) => `
-  background-color: ${theme.palette.primary.main};
-  margin: 16px 8px;
-  `}
+  margin-right: 1.563rem;
+  font-weight: bold;
+  font-size: 1.25rem;
+  color: #ffffff;
+  opacity: 0.87;
```

Comment:
Emotion supports dicts as parameters here which gives us type support. Something like
```
const Title = styled(Typography)({
  marginRight: "1.563rem",
  ...
});

Reply:
updated all to use dicts as parameters